packages/discord.js/src/structures/ApplicationCommand.js

```diff
@@ -127,11 +127,11 @@ class ApplicationCommand extends Base {
   /**
    * Data for creating or editing an application command.
    * @typedef {Object} ApplicationCommandData
-   * @property {string} name The name of the command
-   * @property {string} description The description of the command
-   * @property {ApplicationCommandType} [type] The type of the command
+   * @property {string} name The name of the command, must be in all lowercase if type is CHAT_INPUT
+   * @property {string} description The description of the command, if type is CHAT_INPUT
+   * @property {ApplicationCommandType} [type="CHAT_INPUT"] The type of the command
```

Comment:
I think this default was here for backwards compatibility when v13 released, we can probably get rid of it now or is that not needed?

Reply:
I don't get what you mean sorry

Reply:
The type default can be removed now, as it was there for backwards compatibility but the maintainers should probably say if they want that